src/main/java/net/openhft/chronicle/wire/TextWire.java

```diff
@@ -85,7 +85,7 @@ public class TextWire extends AbstractWire implements Wire {
     static final Supplier<StopCharsTester> END_EVENT_NAME_ESCAPING = TextStopCharsTesters.END_EVENT_NAME::escaping;
     static final Bytes<?> META_DATA = Bytes.from("!!meta-data");
     @Deprecated(/* for removal in x.26, make default true in x.25 */)
-    static final boolean IAE_ON_CNF = Jvm.getBoolean("illegal.argument.for.missing.class.alias", false);
```

Comment:
oh this was already backported?

Reply:
It was, yes - I'd forgotten about it too, but it was the initial backport of the first commit that Even tested and still had the underlying issue.